### PR TITLE
Update infoblox scenario guide

### LINKED ISSUE #73675: The Infoblox dynamic inventory instructions only work under specific circumstances
https://github.com/infobloxopen/infoblox-ansible/blob/master/ansible_collections/infoblox/nios_modules/plugins/inventory/nios_inventory.py

##### SUMMARY

The dynamic inventory script linked from the [Dynamic inventory script section of the Infoblox Guide](https://docs.ansible.com/ansible/latest/scenario_guides/guide_infoblox.html#dynamic-inventory-script) only works when all of the following conditions are true:

 1. the `community.general` Ansible Collection is installed somewhere under the `PYTHONPATH` (this is how the ansible >= 2.10 Python package installs it).
 2. Ansible user uses recent enough Python interpreter that knows how to deal with `__init__.py`-less packages.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
guide_infoblox.rst

##### ADDITIONAL INFORMATION

The `infoblox.nios_modules` Ansible Collection contains an [inventory plugin](https://github.com/infobloxopen/infoblox-ansible/blob/master/ansible_collections/infoblox/nios_modules/plugins/inventory/nios_inventory.py) that offers the same functionality as the "broken" inventory script. So maybe the scenario guide in question should describe how to use that newer plugin instead of the inventory script?

Documentation for Ansible 2.9 is not affected by this, only for Ansible >= 2.10 (because this is when the Python code used in the script moved from core ansible into a collection.

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Replaced Dynamic inventory script details with
Dynamic Inventory plugin details in Infoblox scenario guide.

Fixes: #73675

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/scenario_guides/guide_infoblox.rst

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @justjais Could you please review this? Thanks in advance.

File: docs/docsite/rst/scenario_guides/guide_infoblox.rst [hunk 1]
```diff
@@ -243,50 +243,55 @@ To configure a reverse DNS zone:
             state: present
             provider: "{{ nios_provider }}"
 
-Dynamic inventory script
+Dynamic inventory plugin
 ========================
 
-You can use the Infoblox dynamic inventory script to import your network node inventory with Infoblox NIOS. To gather the inventory from Infoblox, you need two files:
+Ansible provides a dynamic inventory plugin for Infoblox.
```
Comment (felixfontein) [inline docs/docsite/rst/scenario_guides/guide_infoblox.rst]: Ansible does not do that. The Infoblox collection does. As long as infoblox.nios_modules is not included in Ansible, this is an important distinction :)

**Review by justjais — APPROVED** [review]

**Review by samccann — CHANGES_REQUESTED** [review]
Putting this on hold, please don't merge yet.
The infoblox collection is not in Ansible the package and that is what we document right now on docs.ansible.com.  I'll need to request advice from The Powers That Be on what to do here.

**Comment by samccann** [comment]
The Powers that Be came back with a decision. We've written up a 'generic' comment reply that I'll post below and then close this issue. Using the generic comment to get your feedback @Akasurde to see if it's clear enough or needs more detail, for when we hit this situation with another PR from the community.  Thanks for your patience and helping us through this!

**Comment by samccann** [comment]
Thanks @Akasurde for the PR. This content refers to functionality in a collection, and does not belong in the ansible/ansible repository. Can you please move this scenario guide to your collection /docs/ folder? You have two options:

1. Add it as .rst to the docs/docsite/rst sub-folder in the collection repo. This will keep the document with the collection. The reStructuredText content will be published on docs.ansible.com if the collection is included in a future version of Ansible.
2. Convert it to .md and add it directly to the collection /docs folder. The markdown content will appear in Automation Hub if the collection achieves Red Hat certification.

You can do either or both of these and then please open a PR in ansible/ansible to remove the old scenario guide.

**Pull request closed without merging**